### PR TITLE
iOS job-description

### PR DESCRIPTION
**Pull request by marijnschilling**
I copied what we have now, with some minor changes to make it at least not wrong :)

And i added the "what you'll be doing" part, bluntly copied from colin's adroid description 🎉

Let's make it good quick so we can hire the best!

### CODE REVIEW

File: HIRING.md [hunk 1]
```diff
@@ -48,3 +48,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
 
 ## Assignment
 The candidate will be asked to make an assigment with a 24 hour deadline. He'll get to pick the best day to receive the details of the assignment then the clock will start ticking. The assignment will consist in making a simple app to display a list of items that comes from an API. The list should be able to be accessed offline.
+
+# Job description for an iOS position
+
+### A bit about you
+- 01 Experience with building top level applications for Apple iOS/OS X, and started peeping into Watch OS and tvOS.
```
Comment (3lvis) [inline HIRING.md]: Let's keep it simple.
`01 Experience with building top level applications for iOS.`

File: HIRING.md [hunk 2]
```diff
@@ -48,3 +48,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
...
+- 02 Extensive experience with team and client work.
+- 03 A broad understanding of Swift, Core Data and Web-services.
```
Comment (3lvis) [inline HIRING.md]: This could be:

```
03 A broad understanding of Swift, Core Data and RESTful APIs.
```

File: HIRING.md (hunk 1 above)
Comment (marijnschilling) [reply HIRING.md]: I agree, I really don't care if you've started peeping into tv or watch :shrug:

Comment (elland) [reply HIRING.md]: No one cares about tv or watch. 🗡

File: HIRING.md [hunk 3]
```diff
@@ -47,4 +47,26 @@ We’re obviously trying to assess the candidate but remember they are assessing
...
 ## Assignment
-The candidate will be asked to make an assigment with a 24 hour deadline. He'll get to pick the best day to receive the details of the assignment then the clock will start ticking. The assignment will consist in making a simple app to display a list of items that comes from an API. The list should be able to be accessed offline.
+The candidate will be asked to make an assigment with a 24 hour deadline. They will be able to pick the best day to receive the details of the assignment before the clock starts ticking. The assignment will consist of making a simple app to display a list of items that come from an API.
+
+# Job description for an iOS position
+
+### A bit about you
+- 01 Familiarity with building applications for iOS.
+- 02 Teamplayer.
+- 03 Familiarity or interest in Swift, usability and Web services.
```
Comment (elland) [inline HIRING.md]: how about `such as REST and Websockets` here?

**Pull request merged**